src/main/java/com/exactpro/th2/common/event/bean/TreeTable.java

```diff
@@ -16,13 +16,16 @@
 package com.exactpro.th2.common.event.bean;
 
 import com.exactpro.th2.common.event.IBodyData;
+import com.fasterxml.jackson.annotation.JsonInclude;
 
 import java.util.Map;
 
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class TreeTable implements IBodyData {
     public static final String TYPE = "treeTable";
 
     private final String type = TYPE;
+    private String name;
```

Comment:
I think we should make it final and set in the constructor. Also, don't forget to keep the current constructor for backward compatibility

Reply:
Also, please check if the actual name is not blank (if it is `null` it is ok)